pages.pt_BR/common/esbuild.md

```diff
@@ -0,0 +1,36 @@
+# esbuild
+
+> Empacotador e minificador JavaScript construído para velocidade.
+> Mais informações: <https://esbuild.github.io/>.
+
+- Empacota uma aplicação JavaScript e imprime para stdout:
+
+`esbuild --bundle {{caminho/para/arquivo.js}}`
+
+- Empacota uma aplicação JSX de stdin:
+
+`esbuild --bundle --outfile={{caminho/para/saída.js}} < {{caminho/para/arquivo.jsx}}`
+
+- Empacota e reduz uma aplicação JSX com mapas de origem no modo `production`:
+
+`esbuild --bundle --define:{{process.env.NODE_ENV=\"production\"}} --minify --sourcemap {{caminho/para/arquivo.js}}`
+
+- Empacota uma aplicação JSX para uma lista de navegadores separados por vírgulas:
+
+`esbuild --bundle --minify --sourcemap --target={{chrome58,firefox57,safari11,edge16}} {{caminho/para/arquivo.jsx}}`
```

Comment:
```suggestion
`esbuild --bundle --minify --sourcemap --target={{chrome58,firefox57,safari11,edge16}} {{caminho/para/arquivo.jsx}}`
```

Too many `--target` values are listed. ;) It's not critical here but our style guide should define how handle such cases: what maximum count of values is allowed.

Reply:
Hi, should I open a new PR to fix the examples for the english page too ?

Reply:
You can, but I think the current value is good

Reply:
Ok, I'll leave the translation values as they are, to match the english page. Thank you.